cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -44,7 +44,8 @@ const (
 
 	// Deprecated: Use clusterctlv1.CertManagerVersionAnnotation instead.
 	// This is maintained only for supporting upgrades from cluster created with clusterctl v1alpha3.
-	certManagerVersionAnnotation = "certmanager.clusterctl.cluster.x-k8s.io/version"
+	// TODO: Remove once upgrades from v1alpha3 are no longer supported.
+	certmanagerVersionAnnotation = "certmanager.clusterctl.cluster.x-k8s.io/version"
```

Comment:
```suggestion
	certManagerVersionAnnotation = "certmanager.clusterctl.cluster.x-k8s.io/version"
```
I don't think we should change the var name

Reply:
typo - will revert.